docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
+
+With Inago, it's a piece of cake to deploy an Elasticsearch cluster,
+as well as updating it without any downtime or data loss.
```

Comment:
The line breaks feel arbitrary. Sometimes they are sufficient, sometimes not. I would love to see some consistent and reasonable line breaks to shorten lines.